### PR TITLE
Update CHANGELOG for v0.9.4

### PR DESCRIPTION
**Pull request by nrb**
Signed-off-by: Nolan Brubaker <nolan@heptio.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+#### [v0.9.4](https://github.com/heptio/ark/releases/tag/v0.9.4) - 2018-08-10
```
Comment (skriss) [inline CHANGELOG.md]: We'll need to update the date -- let's just hold this PR until the day of release

Comment (nrb) [reply CHANGELOG.md]: Yep, I figured as much

**Comment by nrb** [comment]
@skriss I updated the date to today whenever you're ready to make the release

**Comment by skriss** [comment]
This LGTM; we should also get this into `master`.

**Comment by nrb** [comment]
> This LGTM; we should also get this into master.

Done in #818.

**Pull request merged**